themes/default/content/blog/enhanced-search-intuitive-navigation-new-docs-experience/index.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Enhanced search & Navigation: The new Pulumi Docs experience"
+date: 2023-06-12
+meta_desc:  At Pulumi, we believe in exceptional documentation experiences that help people using Pulumi find what they need quickly and use it successfully.
+meta_image: meta.png
+authors:
+    - susan-evans
+    - anita-trimbur
+    - sean-holung
+tags:
+    - pulumi-releases
+    - features
+    - ux
+---
+
+Engineers spend a lot of time searching documentation for answers. At Pulumi, we believe in exceptional documentation experiences that help people using Pulumi find what they need quickly and use it successfully. Today, we announce an improved Pulumi documentation experience dedicated to all developers.
+<!--more-->
+
+We prioritized search and navigation, so you can easily browse and search for precisely what you want. Enjoy faster and more accurate searches, with lovely user experience touches to guide you to the right place. We also planned for the future, using design patterns that will grow with content and concepts.
+
+![Pulumi docs welcome banner](imgs/banner.png)
+
+Let's showcase a few types of Pulumi documentation:
+
+## Quickstart experience
```

Comment:
Could we have a section on "Unified navigation" to highlight the consistent top and left navigation that has been introduced across docs and packages content?